plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveMetadata.java

```diff
@@ -375,6 +377,21 @@
 
     public static final String MODIFYING_NON_TRANSACTIONAL_TABLE_MESSAGE = "Modifying Hive table rows is only supported for transactional tables";
 
+    private static final Set<String> TABLE_PROPERTIES_TO_SKIP = ImmutableSet.of(
+            PRESTO_VERSION_NAME,
+            PRESTO_QUERY_ID_NAME,
+            BUCKETING_VERSION,
+            "EXTERNAL",
+            "numFiles",
+            "totalSize",
+            "last_modified_time",
+            "transient_lastDdlTime",
+            "last_modified_by",
+            "STATS_GENERATED_VIA_STATS_TASK",
+            "COLUMN_STATS_ACCURATE",
+            "numRows",
+            "rawDataSize");
```

Comment:
Should we introduce another field in `Table` which captures the these properties to be filtered ?

Reply:
I'm against the Guice option for now. Let's keep it simple

Reply:
Just thinking out loud. Wouldn't it be better/safer to whitelist rather than blacklist ?
I mean list properties we allow to read (as we know them and expect what they bring) rather that list properties to skip, but if we forget something or something new would be introduced we will blindly allow it ?

Reply:
But how would we handle them if an arbitrary properties is added by a different system like hive or spark then we might not be able to list them.